scripts/check_compliance.py

```diff
@@ -1204,15 +1187,16 @@ def _main(args):
         suite = TestSuite("Compliance")
 
     for testcase in ComplianceTest.__subclasses__():
-        test = testcase(suite, args.commits)
-        if args.module:
-            if test._name not in args.module:
-                continue
-        elif test._name in args.exclude_module:
-            print("Skipping " + test._name)
+        if args.module and testcase.name not in args.module:
+            continue
+
+        if testcase.name in args.exclude_module:
+            print("Skipping " + testcase.name)
```

Comment:
While simplifying this, could you add a one-line comment mentioning that "module" and "testcase.name" are the same thing? (short of actually renaming them the same)

It took me almost a long minute to realize they're the same and that this code does nothing complicated after all.

Reply:
Comment added.